config.json.example

```diff
@@ -3,5 +3,5 @@
   "hidserv": {
     "hostname": "avgfxawdn6c3coe3.onion",
     "key": "Il8Xas7uf6rjtc0LxYwhrx"
-  }
```

Comment:
Having a comma here results in an error for `sd-proxy-buster-template` (see [1] below). Removing the comma resolves.



[1]
```
sudo qubesctl --show-output --skip-dom0 --targets sd-proxy-buster-template,sd-proxy state.highstate
sd-proxy-buster-template:
      - Rendering SLS 'base:sd-proxy-template-files' failed: Jinja syntax error: Unable to load json from {
          "submission_key_fpr": "65A1B5FF195B56353CC63DFFCC40EF1228271441",
          "hidserv": {
            "hostname": "<something>.onion",
            "key": "hunter2"
          },
        }; line 51
        
        ---
        [...]
            - pkgs:
              - securedrop-proxy
            - require:
              - sls: fpf-apt-test-repo
        
        {% import_json "sd/config.json" as d %}    <======================
        
        install-securedrop-proxy-yaml-config:
          file.managed:
            - name: /etc/sd-proxy.yaml
            - source: salt://sd/sd-proxy/sd-proxy.yaml
        [...]
        ---
sd-proxy: SKIP (nothing to do)
```
failed: Jinja syntax errror: Unable to load json from { <config is here>